projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -173,7 +173,7 @@ function fetch_subscriber_counts() {
 	$subs_count = 0;
 	if ( is_wpcom() ) {
```

Comment:
Here's some optional feedback

```suggestion
	if ( is_wpcom() ) {
	         $include_paid_subscribers = true;
```